src/mixins/filing-template-mixin.ts

```diff
@@ -1094,20 +1099,20 @@ export default class FilingTemplateMixin extends DateMixin {
     })
 
     // store Folio Number
-    // FUTURE: should we store correction.folioNumber instead?
-    this.setFolioNumber(entitySnapshot?.authInfo?.folioNumber || '')
+    // FUTURE: should we store restoration.folioNumber instead?
+    if (entitySnapshot?.authInfo?.folioNumber) {
+      this.setFolioNumber(entitySnapshot?.authInfo?.folioNumber)
+    }
 
-    // if Transactional Folio Number was saved then store it
+    // store Transactional Folio Number
     if (filing.header.isTransactionalFolioNumber) {
       this.setTransactionalFolioNumber(filing.header.folioNumber)
     }
 
     // store Document Optional Email
-    this.setDocumentOptionalEmail(filing.header.documentOptionalEmail || '')
-
-    // store File Number and POA
-    this.setFileNumber(filing.restoration.courtOrder?.fileNumber)
-    this.setHasPlanOfArrangement(filing.restoration.courtOrder?.hasPlanOfArrangement)
```

Comment:
We already store the court order object on line 436, and we don't care about POA.